docs/content/build.md

```diff
@@ -1,46 +1,54 @@
 **Warning:** The [alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../README.md#recommended-use).
 
------
-## Building your workers
+---
 
-Before running a [deployment of your game](deploy.md), you need to:
-1. Build the bridge and launch configuration of your [workers](workers.md)
-2. Prepare the build configuration of your workers
-3. Build your workers
+## How to build your game
 
-To build your workers, you don't need to know what the bridge and launch configurations are, but if you do want to know more about them, see the SpatialOS documentation on [bridge configuration](https://docs.improbable.io/reference/latest/shared/worker-configuration/bridge-config#bridge-configuration) and [launch configuration](https://docs.improbable.io/reference/latest/shared/worker-configuration/launch-configuration#worker-launch-configuration).
+Before running a [deployment of your game](deploy.md) locally or in the cloud, you need to:
+
+1. Build your workers.
+2. Build the bridge configuration and launch configuration of your [workers](workers.md).
+3. Prepare the build configuration of your workers.
+
+### Building your workers
+
+To build your workers:
+
+* With your project open in the Unity Editor, select **SpatialOS** > **Build `WorkerName` for local** or **Build `WorkerName` for cloud** where `WorkerName` is the name of any worker you want to build.
+
```

Comment:
We would usually use "<WorkerName>" in this context. I think it was a mash-up of two styles in the previous version of this text.

Reply:
Do you mean that it should be:

>"WorkerName"

instead of
>`WorkerName`
?

Reply:
Couldn't do this "<" and ">".

Reply:
I can't show you because GitHub formats it and makes it disapper...<like this>.

Reply:
It just disappeared.

Reply:
This:
`<WorkerName>`
But without the backticks.

Reply:
It's really frustrating not to be able to just write it.

Reply:
I've removed the backticks to illustrate that this stops `<WorkerName>` rendering at all:

https://github.com/spatialos/UnityGDK/blob/399b41b24e3f18ac5ba34cdacc51b7c6b15b70a0/docs/content/build.md

Reply:
I'm unsure what presentation you want here, please clarify.